### DISCUSSION TITLE
Türkçe dil gitti İngilizce oldu meizu 20 pro

### DISCUSSION THREAD

**Discussion by emrekayhan07** [bmax121/APatch#480] [Q&A]
Telefonum Türkçeydi yanlışlıkla İngilizce oldu ve tekrar Türkçe nasıl yaparım yardımcı olur musunuz? Telefonum meizu 20 pro 
![S40529-16500886_com android settings](https://github.com/bmax121/APatch/assets/171157546/e606db2f-3c18-45cf-b374-b525c4699924)

**Comment by pomelohan** [comment]
English Please, Can't understand your language
